packaging/package.sh

```diff
@@ -30,8 +30,14 @@ function signAppBundle() {
   fi
 
   frameworks=$(find "$1"/Contents/Frameworks -depth -type d -name "*.framework" -or -name "*.dylib" -or -type f -perm +111)
-  echo "${frameworks}" | xargs -t -I % codesign --deep --sign "${CODESIGN_IDENTITY}" --options runtime % || true
-  codesign --deep --sign "${CODESIGN_IDENTITY}" --options runtime --force --entitlements "${entitlements}" "$1"
+  echo "${frameworks}" | xargs -t -I % codesign --deep --sign "${CODESIGN_IDENTITY}" --options runtime --timestamp % || true
+
+  # sign the .app bundles inside $1/Contents/Frameworks
+  frameworks=$(find "$1"/Contents/Frameworks -depth -type d -name "*.app" -perm +111)
```

Comment:
We have `-name "*.framework" -or -name "*.dylib"` in the first `frameworks` assignment, I don't know if the `--force` is required here, but maybe we could change the first line to `-name "*.framework" -or -name "*.dylib" -or -name "*.app"` ? More details in the log of this commit explaining why this is needed / what issue it's causing if we don't do that would be useful.

Reply:
We have to sign inside out, so first the `*.framework` and `*.dylib` inside of the `*.app` and then at the end sign the outermost thing, that is the `.app` which is the reason they are separately done and also we don't want to rewrite signatures on the `*.framework` and `*.dylib` files